modules.d/35network-connman/module-setup.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# called by dracut
+check() {
+    require_binaries sed grep connmand connmanctl connmand-wait-online || return 1
```

Comment:
You are still missing a requirement on netroot which is called in cm-run.sh

Reply:
See my previous comments. There is no "netroot" binary anywhere which is installed by the "40network" module.
Neither of the `35network-*` modules need it as a required binary.

Reply:
```
$ grep netroot modules.d/??network*/module-setup.sh
modules.d/40network/module-setup.sh:    inst_script "$moddir/netroot.sh" "/sbin/netroot"
$
```

Reply:
@johannbg I just added a new commit to add netroot as a required binary for every `network-*` module. If it's needed for network-connman, then it won't break CI for network-legacy and network-manager.

Reply:
@johannbg As I suspected, it did break CI for every test using network-legacy or network-manager. I will drop the second commit.

Reply:
Yeah you should not have added netroot as a requirement for every network-* module since each module has it's own requirements and meta modules should not be used for somekind of "shared" dependencies since meta module and modules listed in meta modules should work independendly of the meta module itself and contain everything necessary to work properly by themselves.

btw somethin I missed in the review can you drop `network-` from the module name and simply call the module connman so it's in line with network-manager and systemd-networkd.

Reply:
I renamed the module to "connman".

Reply:
Yup it's necessary and dont forget to update the commit message in the process ( feat(connman): ... )

Reply:
Then I will copy the netroot.sh script from the network module and install that if that's what you want. Keep in mind that connman will be the sole module doing this. None of systemd-networkd, network-legacy, network-wicked or network-manager needs anything, they rely on the upper level network module. Why do you want connman to be the odd one out?

Reply:
I copied netroot.sh from 40network and install it from the connman module now.